application/wlp-web-java-spring/src/main/java/webapp/tier/util/CreateId.java

```diff
@@ -1,35 +1,38 @@
 package webapp.tier.util;
 
+import java.security.NoSuchAlgorithmException;
 import java.util.Random;
 
-public class CreateId {
+public final class CreateId {
```

Comment:
Codacy found an issue: [The utility class name 'CreateId' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6170659)